pkg/broker/broker.go

```diff
@@ -1002,9 +1002,9 @@ func (a AnsibleBroker) LastOperation(instanceUUID uuid.UUID, req *LastOperationR
 
 		if async, provision: it should create a Job that calls apb.Provision. And write the output to etcd.
 	*/
-	a.log.Debug(fmt.Sprintf("service_id: %s", req.ServiceID)) // optional
-	a.log.Debug(fmt.Sprintf("plan_id: %s", req.PlanID))       // optional
-	a.log.Debug(fmt.Sprintf("operation:  %s", req.Operation)) // this is provided with the provision. task id from the work_engine
+	a.log.Debugf("service_id: %s", req.ServiceID)
```

Comment:
Thanks for killing the fmt.Sprint, there's a lot of that left over from the old days :)